packages/core-typings/src/AppOverview.ts

```diff
@@ -0,0 +1,115 @@
+import type { AppStatus } from '@rocket.chat/apps-engine/definition/AppStatus';
+
+import type { App, AppSubscriptionStatus } from './Apps';
+
+export type AppOverview = {
+	appId: string;
+	latest: Latest;
+	isAddon: boolean;
+	addonId: string;
+	isEnterpriseOnly: boolean;
+	isBundle: boolean;
+	bundledAppIds: any[];
+	bundledIn: bundledIn[];
+	isPurchased: boolean;
+	isSubscribed: boolean;
+	subscriptionInfo: SubscriptionInfo;
+	price: number;
+	purchaseType: string;
+	pricingPlans: PricingPlan[];
+	isUsageBased: boolean;
+	createdAt: string;
+	modifiedAt: string;
+	iconFileContent?: string;
+	marketplaceVersion?: string;
+	marketplace?: unknown;
+};
+
+export type bundledIn = {
+	bundleId: string;
+	bundleName: string;
+	apps: App[];
+	addonTierId: string;
+};
+
+export type Latest = {
```

Comment:
@rique223 as I understood, the "latest" version could be different from workspaces. It depends on which engine version they are using. So, what do you think about renaming this object to CurrentVersion? This way we do not assume we are always using the Latest version of an app.

Reply:
Solved